goals.md

```diff
@@ -1,19 +1,23 @@
 # Goals for the EUREC4A metadata repository
 
-In order to find a good design for the metadata structure, a set of goals which should be accomplished with the metadata must be established.
-Based on these goals and available information about platforms, instruments and the like, a prototype for the metadata structure will be established.
-This document is explicitly meant to be updated, comments are always welcome.
+To design the metadata structure, a set of goals that this metadata should serve is to be established. Based on these goals and available information about platforms, instruments and the like, a prototype for the metadata structure will be established. This document is explicitly meant to be updated, comments are always welcome.
 
 ## direct goals
 The achievement of these goals should be easily decidable by a yes / no answer.
 
-* get a complete list of platform <-> PI associations
-* list of measured quantities
-* contact information for data usage
-* provide unique names / ids to platforms and instruments to facilitate data exchange and communication
-* source for campaign definitions (e.g. flight circles)
-* produce an [intake](https://intake.readthedocs.io/en/latest/index.html) catalog for EUREC4A
-* produce [GCMD Keywords](https://earthdata.nasa.gov/earth-observation-data/find-data/gcmd/gcmd-keywords). See also [here](https://gcmd.nasa.gov/search/Keywords.do#keywords) and [here](https://earthdata.nasa.gov/earth-observation-data/find-data/gcmd/gcmd-keywords)
+* establish ownership and responsibility for measurements, e.g.,
```

Comment:
```suggestion
* establish ownership and responsibility for measurements, instruments and platforms, e.g.,
```
It may be useful to create the references mentioned below not only for measurements, but also for platforms. In the end, this goal may be achieved by removing the strong separation between the concepts of `platform`, `instrument` and `measurement`.